waspc/examples/todoApp/ext/queries.js

```diff
@@ -14,6 +14,12 @@ export const getTasks = async (args, context) => {
   return tasks
 }
 
+export const getNumTasks = async (args, context) => {
+  const Task = context.entities.Task
+  const numTasks = await Task.count()
+  return numTasks
```

Comment:
We could we wild and put it all into one line: `return context.entities.Task.count()`

Reply:
done